packages/venia-ui/lib/components/Checkout/Receipt/receipt.js

```diff
@@ -5,15 +5,16 @@ import Button from '../../Button';
 import defaultClasses from './receipt.css';
 
 const Receipt = props => {
-    const { createAccount, history, reset, user } = props;
+    const { createAccount, history, reset, onClose, user } = props;
 
     const classes = mergeClasses(defaultClasses, props.classes);
 
     useEffect(() => reset, [reset]);
 
     const handleCreateAccount = useCallback(() => {
+        onClose();
```

Comment:
Yea, you're right. I'm not sure why I did this. I'll double check and respond.

Reply:
Ok so what happens here is that the user clicks `Create Account` and we navigate them, without reloading, to `/create-account`. The order here but it is necessary to call `onClose` so that the next time we render the cart it is on the correct step.

TL;DR - I'll move this after the `createAccount` call.